### DISCUSSION TITLE
Hesabı kurtarma çerezleri geri getirme.

### DISCUSSION THREAD

**Discussion by Abdullahoda** [IBAX-io/go-ibax#1332] [Soru Cevap - Türkçe] [answered]
Google.com/history buradan geçmişe bakabilirsiniz sayfayı bulabilirseniz hesabınız açılabilir Google den arastırın cerezleri geri getirme diye..

**Comment by mardi16** [comment]
en güzeli unut gitsin :)

**Comment by Yusufknk** [comment, marked as answer]
Eski cüzdanın kelimelerini yeni cüzdan oluştur kısmından girerek eski cüzdanınıza erişim sağlayabilirsiniz.
